wpilibj/src/main/java/edu/wpi/first/wpilibj/geometry/Twist2d.java

```diff
@@ -0,0 +1,34 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj.geometry;
+
+/**
+ * A movement along an arc at constant curvature and velocity. We can use ideas
+ * from differential calculus to create new Pose2ds from a Twist2d and vise
+ * versa.
+ *
+ * <p>A Twist can be used to represent a difference between two poses, a velocity,
+ * an acceleration, etc.
+ */
+@SuppressWarnings("MemberName")
+public class Twist2d {
+  /**
+   * Linear "dx" component.
+   */
+  public double dx = 0.0;
```

Comment:
Why no getters and setters like you've done for other classes?

Reply:
Given that this is a POD, [this is probably ok](https://www.oracle.com/technetwork/java/codeconventions-137265.html#177).

Reply:
Are there other POD classes to compare to?